doc/man3/EC_GROUP_copy.pod

```diff
@@ -181,9 +181,17 @@ EC_GROUP_method_of returns the EC_METHOD implementation in use for the given cur
 
 EC_GROUP_get0_generator returns the generator for the given curve or NULL on error.
 
-EC_GROUP_get_order, EC_GROUP_get_cofactor, EC_GROUP_get_curve_name, EC_GROUP_get_asn1_flag, EC_GROUP_get_point_conversion_form
-and EC_GROUP_get_degree return the order, cofactor, curve name (NID), ASN1 flag, point_conversion_form and degree for the
-specified curve respectively. If there is no curve name associated with a curve then EC_GROUP_get_curve_name will return 0.
+EC_GROUP_get_order() returns 0 if the order is not set (or set to zero) and 1 otherwise.
+
+EC_GROUP_get_cofactor() returns 0 if the cofactor is not set (or is set to zero) and 1 otherwise.
+
+EC_GROUP_get_curve_name() returns the curve name (NID) for the specified curve or will return 0 if no curve name is associated.
```

Comment:
I think it would be better to say that it returns `NID_undef` in the latter case

Reply:
Also, you could replace "`for the specified curve`" with "`for B<group>`", referring to the argument name in the `SYNOPSIS` entry.

In general we should try to avoid mixing group/curve as it could create confusion in programmers reading the API documentation.